### ISSUE TITLE
Error when using CSS3's var() in rgba()

### ISSUE THREAD

**Issue by philSixZero** [dlmanning/gulp-sass#658]
## Description
When calling rgba() and passing a CSS3 variable, gulp-sass results in an error:
```css
:root {
  --color: 0,0,0,.5; /* Black with an alpha of 50% */
}

div {
  border-color: rgba(var(--color)); /* CSS3 is able to do this, gulp-sass throws an error */
}
```
### Result
**Error:** overloaded function 'rgba' given wrong number of arguments

## Conclusion
Guess this is a :bug:

**Comment by xzyfer** [comment]
This is not a big. The CSS rgba function takes 4 arguments (r,g,b,a). The
Sass rgba function allows a 2 argument syntax (Color,a).

Color is a Sass type that represents a CSS color. You're trying to mix
these two very different syntaxes which is correctly resulting in an error.

**Comment by allen-garvey** [comment]
See https://github.com/sass/node-sass/issues/2251. I had the same problem, but when I tried the workaround suggested (use RGBA instead of rbga) I was able to get it to work.

**Comment by xzyfer** [comment]
Fixed in 4.8.0
